client/src/pages/board/components/dashboardUI/SaveBoardButton.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Button} from '@material-ui/core';
+import {saveBoard} from '../../../../api/Board';
```

Comment:
lol this path

Reply:
I was literally going to mention setting up absolute paths on day 1 but I kept forgetting and after a while I just let it go. I just wasn't sure if it would be well integrated within editors (e.g. auto imports) and eslint/prettier (format on save) but looks like it might be worth looking into.

Reply:
Yeah. Definitely not a priority but in the long run it would be helpful on a project.

Reply:
I lol'ed when I was importing the path. I'll open a separate issue for absolute paths.